src/main/java/mate/academy/hw01/Recursia.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.hw01;
+
+public class Recursia {
+    public static void main(String[] args) {
+        System.out.println("Fact(4) = " + fact(4));
+        System.out.println("Fibo(7) = " + fibo(7));
+        System.out.println("CosTaylor(3,3) = " + getCosTaylor(3, 3));
+        System.out.println("SinTaylor(4,4) = " + getSinTaylor(4, 4));
+    }
+
+    private static int fact(int num) {
+        if (num == 0) {
+            return 1;
+        }
+        return fact(num - 1) * num;
+    }
+
+    private static int fibo(int num) {
+        if (num == 1 || num == 2) {
+            return 1;
+        }
+        return fibo(num - 1) + fibo(num - 2);
+    }
+
+    private static double getCosTaylor(double y, double x) {
+        if (y > 999) {
```

Comment:
move hardcode to constants, and what 999 means here ?

Reply:
if "y" a very large number, the result of the method approaches to 1

Reply:
move this to constant and and ability to pass this  to method. Because for you 999 is large for other user of your method it may be not.

Reply:
for now just leave this like it is